gnat2goto/driver/tree_walk.adb

```diff
@@ -5110,7 +5124,7 @@ package body Tree_Walk is
             Append_Op (Block, Do_Loop_Statement (N));
 
          when N_Block_Statement =>
-            Warn_Unhandled_Construct (Statement, "block");
+            Append_Op (Block, Do_N_Block_Statement (N));
```

Comment:
Why not append `do_subprogram_or_block` directly?

Reply:
Interface. Wrapper. Consistent with others.  If impl. changes in future, this does not change.